go.mod

```diff
@@ -41,6 +42,7 @@ require (
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
 	github.com/onsi/ginkgo/v2 v2.17.2
 	github.com/onsi/gomega v1.33.1
+	github.com/pkg/errors v0.9.1
```

Comment:
This package is deprecated, but I assume this comes from one of our dependencies?